bundles/org.openhab.binding.adorne/src/main/demo/items/demo.items

```diff
@@ -0,0 +1,5 @@
+Switch LightBathroom {channel="adorne:switch:home:bathroom:power"}
```

Comment:
Short of having automated tests I use these for manual testing. Since tests get checked in I felt these files should be checked in as well. 
No change made.

Reply:
If it's needed for tests, move to  `src/test/resources`